cookbooks/aws-parallelcluster-install/recipes/base.rb

```diff
@@ -195,6 +195,14 @@
   mode "0644"
 end
 
+# Disable DLAMI multi eni helper
```

Comment:
Shall we generalize the disable_log4j_patcher recipe to something like disable_services and add this there so that we don't overload the base recipe?

Reply:
Done